source/simulator/assembly.cc

```diff
@@ -2233,7 +2273,26 @@ namespace aspect
                                 std_cxx11::_5));
     assembler_objects.push_back (std_cxx11::shared_ptr<internal::Assembly::Assemblers::AssemblerBase<dim> >
                                  (complete_equation_assembler));
+    assembler_objects.push_back (std_cxx11::shared_ptr<internal::Assembly::Assemblers::AssemblerBase<dim> >
+                                 (melt_equation_assembler));
```

Comment:
does this not have to be guarded by `if (parameters.include_melt_transport)`? If not, why not?